modules/application/src/main/java/edu/northeastern/truthtree/controller/basicInfo/IBasicInfo.java

```diff
@@ -17,25 +22,25 @@
    * @param range The start and end values that will be used to filter the states returned.
    * @return JSONArray that contains states that are within the provided range.
    */
-  JSONArray getBasicStatesInfo(@RequestParam
-      (value = POPULATION_RANGE, required = false)
-      int[] range);
+  ResponseEntity<PageDTO<LocationDTO>> getBasicStatesInfo(
+          @RequestParam(value = POPULATION_RANGE, required = false) int[] range,
```

Comment:
Can you rename page to pageNumber? I guess other pagination supported apis have used pageNumber.

Reply:
addressed